canonical-web-and-design/ubuntu.com
# Add canonical urls for maas blog posts

## Done

- Add canonical urls for maas blog posts

## QA

- Go to /blog/announcing-maas-3-1
- View source code
- Check that `<link ref="canonical"` href is pointing to maas.io

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4882

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
